lib/index.js

```diff
@@ -112,4 +112,33 @@ module.exports = class {
       this.methods.GET,
       `${this.base}taxonomy_term.${this.format}?${qs.stringify(filterOptions, { indices: false })}`);
   }
+  /**
+   * getRelatedComments
+   * Related Comments Request wrapper.
+   * @param  {Promise} data
+   *   Comment IDs from a node query.
+   * @return {Promise}
+   *   The response from the Drupal.org HTTP api.
+   */
+  getRelatedComments(data) {
+    return Array.isArray(data.list[0].comments) ? Promise.all(data.list[0].comments
+      .map(comment => this.comment({ cid: comment.id })))
+      .then(commentEntities => commentEntities.map(commentEntity => commentEntity.list[0])) :
+      Promise.reject('No Comments available.');
+  }
+  /**
+   * getRelatedTaxonomyTerms
+   * Related Taxonomy Term Request wrapper.
+   * @param  {Promise} data
+   *   Taxonomy IDs from a node query.
+   * @return {Promise}
+   *   The response from the Drupal.org HTTP api.
+   */
+  getRelatedTaxonomyTerms(data) {
+    const taxonomyTermsArray = data.list[0].taxonomy_vocabulary_9;
```

Comment:
I think we had talked about only accepting an array of `nid`'s? That way we don't have to even think about the format of the data, and then just check that the length of the array passed is greater than `0` and some basic sanity checks around the contents.